data/mods/Magiclysm/Spells/attunements/Ice_Elemental.json

```diff
@@ -22,5 +22,46 @@
     "min_duration": 18000,
     "max_duration": 120000,
     "duration_increment": 4500
+  },
+  {
+    "id": "cold_heart",
+    "type": "SPELL",
+    "name": "Cold Heart",
+    "description": "Harshly decrease temperature of your enemies from inside, shattering his organs and slowing down his bloodstream.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "NO_PROJECTILE", "SILENT" ],
+    "effect": "attack",
+    "shape": "blast",
+    "damage_type": "cold",
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 1.5,
+    "min_range": 6,
+    "max_range": 20,
+    "range_increment": 0.35,
+    "max_level": 40,
+    "difficulty": 10,
+    "spell_class": "ICE_ELEMENTAL",
+    "base_casting_time": 150,
+    "final_casting_time": 75,
+    "casting_time_increment": -1.9,
+    "energy_source": "MANA",
+    "base_energy_cost": 80,
+    "extra_effects": [ { "id": "cold_heart_slow" } ]
+  },
+  {
+    "id": "cold_heart_slow",
+    "type": "SPELL",
+    "name": "Cold Heart slowdown",
+    "description": "Give slowdown for Cold Heart spell. Subspell you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "blast",
+    "effect_str": "cold_heart_effect",
+    "max_level": 40,
```

Comment:
```suggestion
```
N/A for subspells, as far as I am aware.

Reply:
need to check it, but afaik it scale with main spell level, but if you dont specify it, it will be alvays in 1 lvl, that can break some scale stuff, like dutation scale in this case

Reply:
I think it takes it from the original spell - e.g., `biomancer_visceral_paralyze` in biomancer.json. (Or is that not working right?)

Reply:
Yep, i was right. As an examle: pyroclastic flow skill that have scalable subspell that create smoke - first one is default, with `max_level`
![image](https://user-images.githubusercontent.com/67688115/110253880-d3c53d80-7f8c-11eb-9d8d-60e28e18b342.png)
and second is same, but without "max_level"
![image](https://user-images.githubusercontent.com/67688115/110253895-eb042b00-7f8c-11eb-94ec-f54c0273a809.png)
Summary - its possible to delete level from subspell, if it doesnt have any scale inside, otherwise you must left it for correct scaling

Reply:
I'll try to check further into this (and any current subspells that may need such), probably Tuesday.